transformerlab/routers/data.py

```diff
@@ -53,18 +53,20 @@ async def dataset_gallery() -> Any:
 @router.get("/info", summary="Fetch the details of a particular dataset.")
 async def dataset_info(dataset_id: str):
     d = await db.get_dataset(dataset_id)
-
     if d is None:
         return {}
-
     r = {}
     # This means it is a custom dataset the user uploaded
     if d["location"] == "local":
         dataset = load_dataset(path=dirs.dataset_dir_by_id(dataset_id))
         # print(dataset['train'].features)
         r["features"] = dataset["train"].features
     else:
-        ds_builder = load_dataset_builder(dataset_id)
+        dataset_config = d.get("json_data").get("dataset_config")
```

Comment:
You can probably just do d["json_data"] if you are assuming it is present (or add {} as a second parameter if you want to use this to guard against a missing json_data parameter).
But then what you can do in the second get is make the second parameter None to make it clear that's what you get if dataset_config doesn't exist?